servant-js/servant-js.cabal

```diff
@@ -59,16 +59,16 @@ executable counter
   else
     buildable: False
 
-  build-depends:
-      aeson
-    , base
-    , filepath
-    , servant == 0.5.*
-    , servant-server == 0.5.*
-    , servant-js == 0.5.*
-    , stm
-    , transformers
-    , warp
+  build-depends:    aeson
+                  , base
+                  , filepath
+                  , lens >= 4
+                  , servant
+                  , servant-server
+                  , servant-js
+                  , stm
+                  , transformers
```

Comment:
We've taken to upper and lower bounds. Lower bounds because that's the only honest thing to do. Upper bounds because Snoyman et al. notify us when something outside of them is release, making it very easy to maintain them.